google/cloud/storageinsights/quickstart/quickstart.cc

```diff
@@ -23,13 +22,15 @@ int main(int argc, char* argv[]) try {
     return 1;
   }
```

Comment:
```
  if (argc != 3) {
    std::cerr << "Usage: " << argv[0] << " project-id location-id\n";
    return 1;
  }
```

Reply:
I think I fixed this in commit: `"oops"` https://github.com/googleapis/google-cloud-cpp/pull/11339/commits/db419e21e44176c189f260b6a13f307d0f8fa93b

Reply:
Sorry ... I must have failed to scroll down to the last commit.